rust/fastsim-core/src/lib.rs

```diff
@@ -29,6 +29,7 @@
 //! ```
 
 extern crate ndarray;
+extern crate tempfile;
```

Comment:
you probably don't need this line.  The `extern` syntax has been unneeded since a much earlier version of rust.  See what happens if you delete it.